src/firstvoices/backend/models/dictionary.py

```diff
@@ -140,13 +142,44 @@ class TypeOfDictionaryEntry(models.TextChoices):
         related_name="incoming_related_dictionary_entries",
     )
 
+    # from fvcharacter:related_words
+    related_characters = models.ManyToManyField(
+        Character,
+        blank=True,
+        through="DictionaryEntryRelatedCharacter",
+        related_name="dictionary_entries",
+    )
+
     class Meta:
         verbose_name = _("Dictionary Entry")
         verbose_name_plural = _("Dictionary Entries")
 
     def __str__(self):
         return self.title
 
+    def save(self, *args, **kwargs):
+        self.clean_confusables()
+        self.set_custom_order()
```

Comment:
It might be an idea to do a check here and not call these functions if confusables don't exist for a site.